client/overview/task-list/dispute-tasks.ts

```diff
@@ -0,0 +1,221 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import moment from 'moment';
+import { getHistory } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import type { TaskItemProps } from './types';
+import type { CachedDispute } from 'wcpay/types/disputes';
+import { formatCurrency } from 'wcpay/utils/currency';
+import { getAdminUrl } from 'wcpay/utils';
+import wcpayTracks from 'wcpay/tracks';
+
+/**
+ * Returns true if the dispute is due within the specified number of days.
+ *
+ * @param {CachedDispute} dispute - The dispute to check.
+ * @param {number} [days] - The number of days to check. Defaults to 1.
```

Comment:
If you agree that the 2nd argument does not need to be optional...

```suggestion
 * @param {number} days - The number of days to check.
```